src/NodaTime/Calendars/HebrewYearMonthDayCalculator.cs

```diff
@@ -159,26 +159,24 @@ internal override YearMonthDay AddMonths(YearMonthDay yearMonthDay, int months)
             return new YearMonthDay(year, month, day);
         }
 
-        // Note to self: this is (minuendInstant - subtrahendInstant) in months. So if minuendInstant
-        // is later than subtrahendInstant, the result should be positive.
-        internal override int MonthsBetween(YearMonthDay minuendDate, YearMonthDay subtrahendDate)
+        internal override int MonthsBetween(YearMonthDay start, YearMonthDay end)
         {
             // First (quite rough) guess... we could probably be more efficient than this, but it's unlikely to be very far off.
-            int minuendCivilMonth = CalendarToCivilMonth(minuendDate.Year, minuendDate.Month);
-            int subtrahendCivilMonth = CalendarToCivilMonth(subtrahendDate.Year, subtrahendDate.Month);
-            double minuendTotalMonths = minuendCivilMonth + (minuendDate.Year * MonthsPerLeapCycle) / (double) YearsPerLeapCycle;
-            double subtrahendTotalMonths = subtrahendCivilMonth + (subtrahendDate.Year * MonthsPerLeapCycle) / (double) YearsPerLeapCycle;
-            int diff = (int) (minuendTotalMonths - subtrahendTotalMonths);
+            int startCivilMonth = CalendarToCivilMonth(start.Year, start.Month);
+            double startTotalMonths = startCivilMonth + (start.Year * MonthsPerLeapCycle) / (double)YearsPerLeapCycle;
+            int endCivilMonth = CalendarToCivilMonth(end.Year, end.Month);
+            double endTotalMonths = endCivilMonth + (end.Year * MonthsPerLeapCycle) / (double) YearsPerLeapCycle;
+            int diff = (int) (endTotalMonths - startTotalMonths);
 
-            if (Compare(subtrahendDate, minuendDate) <= 0)
+            if (Compare(start, end) <= 0)
```

Comment:
It does, but we don't want to use it - because it might give the wrong answer for the Hebrew calendar because of the month numbering issue.

From the docs of `YearMonthDay`:

```csharp
/// This type is naive: comparisons are performed assuming that a larger month number
/// always comes after a smaller month number, etc.
/// This is suitable for most, but not all, calendar systems.
```

:)